client/src/components/RelatedItems/ProductCard.jsx

```diff
@@ -6,16 +6,19 @@ import Stars from '../SharedComponents/Stars.jsx';
 import utilityFns from '../../utilityFns.js';
 
 function ProductCard({relatedProductInfo}) {
-  const imageSource = relatedProductInfo.style.photos[0].url ? relatedProductInfo.style.photos[0].url : noImageAvailable;
-  const productprice = Number(relatedProductInfo.default_price);
-  let {averageRating} = utilityFns.processReviewMetadata(relatedProductInfo);
+  let {currentProduct,
+    productStylesArray,
+    reviews} = relatedProductInfo;
+  const imageSource = productStylesArray[0].photos[0].url ? productStylesArray[0].photos[0].url : noImageAvailable;
```

Comment:
I prefer using objects to arrays b/c they keys can make the code more readable -- but that's more of a personal preference and I'm not familiar enough w/ your API data to make any recommendations on this

Reply:
The productStyleArray is part of Alex's component data. He was using it as an array in his dummy data, so I kept the same format to make it easier for him to use the data coming from the API. Otherwise, he'd need to change his components.